pkgs/servers/sql/postgresql/ext/timescaledb.nix

```diff
@@ -15,6 +15,10 @@ stdenv.mkDerivation rec {
   };
 
   cmakeFlags = [ "-DSEND_TELEMETRY_DEFAULT=OFF" "-DREGRESS_CHECKS=OFF" "-DTAP_CHECKS=OFF" ]
+    ++ lib.optionals (!stdenv.buildPlatform.canExecute stdenv.hostPlatform)
+    [
+      "-DPG_CONFIG=${buildPackages.postgresql}/bin/pg_config"
```

Comment:
But wouldn't this `pg_config` print values for the buildPlatform postgresql in stead of the hostPlaform one?

Reply:
You are right. A better approach is needed. Closing this PR.